api/v1alpha1/utils.go

```diff
@@ -25,8 +25,12 @@ func GetDriverUpgradePolicy(
 	driverUpgradePolicy.MaxParallelUpgrades = ofedUpgradePolicy.MaxParallelUpgrades
 
 	driverUpgradePolicy.PodDeletion = nil
-	driverUpgradePolicy.WaitForCompletion = getWaitForCompletionSpec(*ofedUpgradePolicy.WaitForCompletion)
-	driverUpgradePolicy.DrainSpec = getDrainSpec(*ofedUpgradePolicy.DrainSpec)
+	if ofedUpgradePolicy.WaitForCompletion != nil {
+		driverUpgradePolicy.WaitForCompletion = getWaitForCompletionSpec(*ofedUpgradePolicy.WaitForCompletion)
```

Comment:
a good improvement is to pass by reference and not copy on stack for both getWait... and getDrain...
but not related to this PR.